sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -1650,6 +1650,7 @@ SUBROUTINE CLDRAD
     !
     !--- Grid-scale cloud occurs when the mixing ratio exceeds QCLDmin
     !    or in the presence of snow when RH>=95% or at/above the PBL top.
+    !    However, for RRFS (FV3R), simply use a threshold cloud fraction. 
```

Comment:
@jaymes-kenyon Can you add a change log with your name at the beginning session of this subroutine?

Reply:
Good catch, @WenMeng-NOAA — all set now.